doc/Makefile.am

```diff
@@ -29,6 +30,16 @@ NABCSRCFILES = GregorioNabcRef.tex veni.gabc
 GregorioRef.pdf: $(SRCFILES)
 	$(MAKE) $(AM_MAKEFLAGS) -C ../src gregorio
 	../src/gregorio -o factus.gtex $(<D)/factus.gabc
+	texlua scorefonttable.lua greciliaeR ../fonts/greciliae.ttf \
```

Comment:
I'm not sure I can find the `scorefonttable.lua` script, did you forget to add it?